tiled/server/authentication.py

```diff
@@ -362,7 +379,10 @@ async def get_current_scopes(
             api_key, settings, request.app.state.authenticated, db
         )
     elif decoded_access_token is not None:
-        return decoded_access_token["scp"]
+        if isinstance(settings.authenticator, ProxiedOIDCAuthenticator):
+            return set(decoded_access_token["scope"].split(" "))
```

Comment:
:+1: According to StackOverflow, [this is the standard format of scopes in an access token](https://stackoverflow.com/a/70184278)